pywinauto/controls/common_controls.py

```diff
@@ -3286,14 +3286,109 @@ class IPAddressWrapper(HwndWrapper.HwndWrapper):
 
 #====================================================================
 class CalendarWrapper(HwndWrapper.HwndWrapper):
-    "Class that wraps Windows Calendar common control "
+    "Class that wraps Windows Calendar common control"
 
     friendlyclassname = "Calendar"
     windowclasses = ["SysMonthCal32", ]
     if sysinfo.UIA_support:
         controltypes = [IUIA().UIA_dll.UIA_CalendarControlTypeId]
     has_title = False
+    #----------------------------------------------------------------
+    def __init__(self, hwnd):
+        "Initialise the instance"
+        #HwndWrapper.HwndWrapper.__init__(self, hwnd)
+        super(CalendarWrapper, self).__init__(hwnd)
+
+    #----------------------------------------------------------------
+    def get_current_date(self):
+        "Get the currently selected date"
+        
+        remote_mem = RemoteMemoryBlock(self)
+        system_date = win32structures.SYSTEMTIME()
+        remote_mem.Write(system_date)
+        
+        res = self.send_message(win32defines.MCM_GETCURSEL , 0, remote_mem)
+        remote_mem.Read(system_date)
+        del remote_mem
+        
+        if res == 0:
+            raise RuntimeError('Failed to get time in Calendar')
+        
+        #year = system_time.wYear
+        #month = system_time.wMonth
+        #day_of_week = system_time.wDayOfWeek
+        #day = system_time.wDay
+        #hour = system_time.wHour
+        #minute = system_time.wMinute
+        #second = system_time.wSecond
+        #milliseconds = system_time.wMilliseconds
+        #return (year, month, day_of_week, day, hour, minute, second, milliseconds)
+        return system_date
+    # Non PEP-8 alias
+    GetCurrentDate = get_current_date
 
+    #----------------------------------------------------------------
+    def set_date(self, year, month, day_of_week, day):
+        "Set the currently selected date"
+        

```

Comment:
An empty line after a doc string should be removed in all such cases. Use triple double quotes for doc strings (`"""Set the currently selected date"""`).